.github/workflows/main.yml

```diff
@@ -172,71 +139,3 @@ jobs:
         with:
```

Comment:
Apparently codecov/codecov-action@v1 s deprecated - see https://github.com/marketplace/actions/codecov

codecov/codecov-action@v3 is the latest.

Reply:
note to myself, fix it in next pr.